public/main.css

```diff
@@ -202,46 +246,150 @@ tbody {
   flex-wrap: nowrap;
 }
 
+#copy-text {
+  align-self: flex-start;
+  margin-top: 60px;
+  margin-bottom: 10px;
+  color: #0C0C0D;
+}
+
 #link {
-  width: 216px;
-  height: 41px;
-  border: 1px solid #979797;
+  width: 480px;
+  height: 56px;
+  border: 1px solid #0297F8;
+  border-radius: 6px 0 0 6px;
+  font-size: 24px;
+  color: #737373;
+  font-family: 'SF Pro Text';
+}
+
+#link:disabled {
+  border: 1px solid #05A700;
+  background: #FFF;
 }
 
 #copy-btn {
-  width: 60px;
-  height: 45px;
-  background: #337FEB;
-  border: 1px solid #979797;
+  width: 165px;
+  height: 60px;
+  background: #0297F8;
+  border: 1px solid #0297F8;
+  border-radius: 0 6px 6px 0;
   color: white;
   cursor: pointer;
+  font-size: 15px;
+  font-family: 'SF Pro Text';
 }
 
 #copy-btn:disabled {
-  background: #47B04B;
+  background: #05A700;
+  border: 1px solid #05A700;
   cursor: auto;
 }
 
+#delete-file {
+  width: 176px;
+  height: 44px;
+  background: #FFF;
+  border: 1px solid rgba(12, 12, 13, 0.3);
+  border-radius: 5px;
+  font-size: 15px;
+  font-family: 'SF Pro Text';
+  margin-top: 50px;
+  margin-bottom: 12px;
+  cursor: pointer;
```

Comment:
perhaps we make a rule for `button` and add some default rules to that too, like this one.